cg/meta/workflow/nf_analysis.py

```diff
@@ -812,6 +836,41 @@ def get_cases_to_store(self) -> list[Case]:
             or self.trailblazer_api.is_latest_analysis_qc(case_id=case.internal_id)
         ]
 
+    def get_genome_build(self, case_id: str) -> GenomeVersion:
```

Comment:
This will raise an error for rnafusion as the genome version is not added to the samples (and break the uploads as the delivery report needs this field 😓 )

Reply:
Good catch! I will overwrite for rnafusion then